CHANGES.md

```diff
@@ -32,6 +23,7 @@ New Languages:
 
 Language Improvements:
 
+- enh(shell) Add support for multiline commands with line continuation `\` (#2861) [Guillaume Grossetie][]
```

Comment:
@joshgoebel I'm confused, this language improvement is not part of the 10.4.0-beta1 right?

Reply:
Correct really they heading should probably change. Betas aren’t the same as releases though they’re rolling so things just keep getting added until we have a release

Reply:
So really the header and should be 10.4 rolling or some such. I’ll give it some thought. I only updated it because we have to update the header for production releases but that’s not true for beta releases so we have more flexibility there.